crates/nu-command/src/filesystem/ls.rs

```diff
@@ -352,6 +352,7 @@ fn should_skip_dir(dir: impl AsRef<Path>) -> bool {
 }
 
 fn is_hidden_dir(metadata_attributes: u32) -> bool {
+    #[cfg(windows)]
```

Comment:
I'm guessing that this should be moved up so that `is_hidden_dir` is only available on windows.

Reply:
Thanks!